tests/e2e/standalone/standalone_test.go

```diff
@@ -359,10 +359,11 @@ func testRun(t *testing.T) {
 		output, err := spawn.Command(daprPath, "run", "--dapr-http-port", "9999", "--", "bash", "-c", "curl -v http://localhost:9999/v1.0/shutdown; sleep 10; exit 1")
 		t.Log(output)
 		require.NoError(t, err, "run failed")
-		assert.Contains(t, output, "Exited App successfully", "App should be shutdown before it has a chance to return non-zero")
+		// It would be ideal to check that the spawned app existed with a non-zero
+		// exit code, however when sending a SIGINT/CTRL-C to curl, it does not.
+		// In the future we could find a program that returns 0 in this scenario.
```

Comment:
You might as well not have have this test at all without this check. The exit 1 is how we tell the difference between "shutdown received and respected" vs "shutdown ignored". 

```
willardstanley@Willards-MacBook-Pro cli % bash -c 'trap "exit 0" SIGINT; sleep 200; exit 12'; echo $? 
^C0
```

Reply:
Litmus test: If you mangle the curl URL, so that the shutdown API doesn't get called, does the test fail like it should?